src/App.tsx

```diff
@@ -1,24 +1,247 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useRef } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import * as postServices from './api/todos';
+import { Todos } from './api/components/todos';
+import { Footer } from './api/components/footer';
+import { Error } from './api/components/error';
+import { Loader } from './api/components/loader';
 
-const USER_ID = 0;
+const USER_ID = 12030;
+
+function getPreparedTodos(todos: Todo[], filter: string): Todo[] {
+  const preparedTodos = todos.filter(todo => {
+    switch (filter) {
+      case 'completed':
+        return todo.completed;
+
+      case 'active':
+        return !todo.completed;
+
+      default:
+        return true;
+    }
+  });
+
+  return preparedTodos;
+}
+
+// function error(er: string) {
```

Comment:
Remove comment. Try to write obvious code with meaningful variable and function naming
Fix all cases